scalding-core/src/main/scala/com/twitter/scalding/ExecutionUtil.scala

```diff
@@ -2,17 +2,27 @@ package com.twitter.scalding
 
 import com.twitter.algebird.Semigroup
 
+import scala.util.{Failure, Success, Try}
+
 object ExecutionUtil {
   /**
+   * Useful for when using parallelism and want to output the final state of the job
+   */
+  def liftedSequenceUnit[T](executions: Execution[Seq[Try[T]]], handleComplete: Try[T] => Unit): Execution[Unit] =
```

Comment:
this API seems really easy to misuse. Not sure it should be public in scalding-core.

1. assumes executions can't fail.
2. uses side effects, and applies the function an unknown number of times (once per item).
3. replaces a somewhat uncommon pattern with only 3 lines fewer of code.